tests/_support/Step/Acceptance/Accounts.php

```diff
@@ -11,37 +11,17 @@ class Accounts extends \AcceptanceTester
      */
     public function createAccount($name)
     {
-        $I = new EditView($this->getScenario());
-        $DetailView = new DetailView($this->getScenario());
-        $Sidebar = new SideBar($this->getScenario());
-        $faker = $this->getFaker();
+        global $db;
```

Comment:
I don't love using the global here, but I couldn't figure out how to get it to work with `DBManagerFactory::getInstance()`. Any ideas?